.circleci/config.yml

```diff
@@ -103,8 +103,103 @@ commands:
           name: Install libvips
           command: sudo apt-get install -y libvips
 
-jobs: {}
+  install_solidus:
+    parameters:
+      flags:
+        type: string
+        default: "# no options"
+        description: "flags to be passed to `bin/rails g solidus:install"
+    steps:
+      - run:
+          name: "Cleanup & check rails version"
+          command: |
+            gem environment path
+            rm -rf /tmp/my_app /tmp/.tool-versions # cleanup previous runs
+
+            ruby -v >> /tmp/.tool-versions
+            gem search -eq rails >> /tmp/.tool-versions # get the latest rails from rubygems
+            gem search -eq solidus >> /tmp/.tool-versions # get the latest solidus from rubygems
```

Comment:
Does this work only with the latest solidus? What happen when we will release v3.3?

Reply:
@kennyadsl I think it will just bust the cache, which is the intended outcome IIRC.